addon/model/model-range.ts

```diff
@@ -259,6 +262,63 @@ export default class ModelRange {
     return this.start.sameAs(this.end);
   }
 
+  normalize(): void {
```

Comment:
could you add a doc comment saying what this actually does?

Reply:
Looking at it, it seems like you're making sure that start is before end. On a range, this should always be the case. Where did you encounter this not being so?



Also, modelPosition.compare exists, most of this function just reimplements the same logic

Reply:
yes I actaully wanted to ask you abou this, in tests this was the case, range end and range start were always based on the order of parameters passed to the constrcutor.